README.md

```diff
@@ -0,0 +1,6 @@
+# Reruns
+
+Reruns is an application that generates random episodes for you when you are rewatching tv shows. It also has a ranking
+system so that your favorite episodes show up more often.
+
+<img src="http://oi67.tinypic.com/ftkcxi.jpg" width="300"> <img src="http://oi67.tinypic.com/wsqsza.jpg" width="300"> 
```

Comment:
Change the pic and use a screenshot for the pop-up you get for an episode. The UI for the main screen is bad.